src/git/zcl_abapgit_git_pack.clas.abap

```diff
@@ -338,7 +338,6 @@ CLASS zcl_abapgit_git_pack IMPLEMENTATION.
           lo_progress TYPE REF TO zcl_abapgit_progress,
           lt_deltas   LIKE ct_objects.
 
-
```

Comment:
there are no changes to the logic in this file, can you help excluding it from the PR?